bundles/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/controls/LxControl.java

```diff
@@ -705,6 +717,20 @@ void removeAllChannels() {
         callbacks.clear();
     }
 
+    /**
+     * Adds a new state to the control.
+     *
+     * @param name name of the state
+     * @param uuid UUID of the state - can be null
+     */
+    private void addState(String name, String uuid) {
+        if (uuid == null) {
+            logger.warn("Unexpected null UUID in state {} inside state array in control {}", name, this.name);
```

Comment:
I'm not sure what this warn message intend is? If the user would see this message what should the user do? If the user can't do anything useful here it should either be debug log or the message should give a clear message what the consequence is when this is null and possible how to (possible) fix it.